r/R/scalar.R

```diff
@@ -44,6 +58,8 @@
 #' Scalar$create(404)
 #' # If you pass a vector into Scalar$create, you get a list containing your items
 #' Scalar$create(c(1, 2, 3))
+#' # If you pass a data.frame with multiple columns into Scalar$create, you get the first row of the data.frame
+#' Scalar$create(mtcars)
```

Comment:
```suggestion
```
Let's leave this out; it's very atypical usage, and not something we'd expect users to need to do.

Reply:
I have no strong reason to insist that this be done, so I will remove it.

Reply:
Cheers!  And thanks in general for looking into these docs issues - these kinds of things make for a much better experience for our users.